test/functions/iterators/diten/iteratorNoYield.good

```diff
@@ -1 +1 @@
-iteratorNoYield.chpl:9: error: illegal use of function that does not return a value: 'itr'
+iteratorNoYield.chpl:9: error: cannot iterate over values of type void
```

Comment:
This is an interesting one.  Should an iterator with no yields be treated as yielding voids, or should it result in the loop collapsing into a no-op?  I might be inclined to do the latter, as it seems like a potentially powerful feature (e.g., if a param conditional folded all the actual yields out of the body...?)